### ISSUE TITLE
"TypeError: rval.replaceAll is not a function" when attempting to build locally

### ISSUE THREAD

**Issue by neilenns** [dementedmonkey/cj4-mcdu#34]
Thanks for checking in the buildNodes.md file with your latest release. I totally hear you when you say: "Notes for myself becuase every time I step away from this for a while I forget the details." Been there many times in my own projects!

Following your steps I'm unable to get a successful build. It always fails with this error from `rollup.config.js`:

```
[!] TypeError: rval.replaceAll is not a function
TypeError: rval.replaceAll is not a function
    at subpath (D:\git\cj4-mcdu\src\app\client\rollup.config.js:18:17)
```

I get this failure building both on Windows and in Ubuntu. I've poked around a bit and can't figure out why it's failing, other than I assume for some reason `rval` is null instead of a string?

**Comment by neilenns** [comment]
Ok, that's not the reason. I added some debug output and rval is really something:

```
D:\git\cj4-mcdu\src\app\client\public\*
```

Did a bit of digging and apparently `replaceAll` isn't available everywhere (and apparently not in whatever version of JS is getting used on my two build environments).

[A suggestion from stackoverflow ](https://stackoverflow.com/a/63958411) is to swap to this:

```javascript
    rval = rval.split('\\').join('/');
```

Which I did and seems to work.
